### PR TITLE
Appended Books and Podcasts section to end of Resources page [Fixes #3181]

### LINKED ISSUE #3181: Include Books/Podcasts in Resources
A discussion about Ethereum/Crypto related books on discord made me wonder why we aren't recommending any. I think a lot of value can be added by pointing users to resources that they can consume away from their computer.

**Describe the solution you'd like**

Create a section on https://ethereum.org/en/learn/ for books and/or podcasts.

**Additional Context**
Some potential books to get us started.

Books
- The Infinite Machine
- Out of the Ether
- The Age of Cryptocurrency
- The Truth Machine
- Digital Gold
- Kings of Crypto

Podcasts
- Into the Ether
- Bankless

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

### PR DESCRIPTION
**Pull request by TrevorFrench**
## Description

Added a section entitled "Books and podcasts" to the learning resources page. Six books and six podcasts were listed with links.
Books reference the author(s) along with the publish date. Podcasts contain a brief description paraphrased from source websites.

## Related Issue

https://github.com/ethereum/ethereum-org-website/issues/3181

### CODE REVIEW

File: src/content/learn/index.md [hunk 1]
```diff
@@ -119,3 +119,23 @@ Critical views of Ethereum and Cryptocurrencies.
 - [The Challenges of Building Ethereum Infrastructure](https://medium.com/@lopp/the-challenges-of-building-ethereum-infrastructure-87e443e47a4b) _Jan 8, 2018 - Jameson Lopp_
 - [Parsimonious Answers to Difficult Questions](https://www.youtube.com/watch?v=GOkSg0BuSdw&feature=youtu.be) _(Video) March 10, 2019 - Rick Dudley_
 - [There’s no good reason to trust blockchain technology](https://www.wired.com/story/theres-no-good-reason-to-trust-blockchain-technology/) _Feb 6, 2019 - Bruce Schneier_
+
+## Books and podcasts {#books-and-podcasts}
```
Comment (corwintines) [inline src/content/learn/index.md:123]: I think its worthwhile throwing the Mastering Ethereum book in here as well for developers: https://github.com/ethereumbook/ethereumbook

Comment (TrevorFrench) [reply src/content/learn/index.md:123]: That's a good one! I appended it to the list of books.

File: src/content/learn/index.md [hunk 2]
```diff
@@ -119,3 +119,24 @@ Critical views of Ethereum and Cryptocurrencies.
...
+- [The Infinite Machine: How an Army of Crypto-hackers Is Building the Next Internet with Ethereum](https://www.amazon.com/Infinite-Machine-Crypto-hackers-Building-Internet/dp/0062886142) _July 14, 2020 - Camila Russo_
+- [The Age of Cryptocurrency: How Bitcoin and the Blockchain Are Challenging the Global Economic Order](https://www.amazon.com/Age-Cryptocurrency-Blockchain-Challenging-Economic/dp/1250081556) _January 12, 2016 - Paul Vigna, Michael J. Casey_
+- [The Truth Machine: The Blockchain and the Future of Everything](https://www.amazon.com/Truth-Machine-Blockchain-Future-Everything/dp/1250114578) _February 27, 2018 - Paul Vigna, Michael J. Casey_
+- [Digital Gold: Bitcoin and the Inside Story of the Misfits and Millionaires Trying to Reinvent Money](https://www.amazon.com/Digital-Gold-Bitcoin-Millionaires-Reinvent/dp/006236250X) _May 24, 2021 - Nathaniel Popper_
+- [Kings of Crypto: One Startup's Quest to Take Cryptocurrency Out of Silicon Valley and Onto Wall Street](https://www.amazon.com/Kings-Crypto-Startups-Cryptocurrency-Silicon/dp/1647820189) _December 15, 2020 - Jeff John Roberts_
```
Comment (minimalsm) [inline src/content/learn/index.md:132]:
```suggestion
- [Out of the Ether: The Amazing Story of Ethereum and the $55 Million Heist that Almost Destroyed It All](https://www.goodreads.com/book/show/55360267-out-of-the-ether) _September 29, 2020 - Matthew Leising_
- [The Infinite Machine: How an Army of Crypto-hackers Is Building the Next Internet with Ethereum](https://www.goodreads.com/en/book/show/50175330-the-infinite-machine) _July 14, 2020 - Camila Russo_
- [The Age of Cryptocurrency: How Bitcoin and the Blockchain Are Challenging the Global Economic Order](https://www.goodreads.com/en/book/show/22174460-the-age-of-cryptocurrency) _January 12, 2016 - Paul Vigna, Michael J. Casey_
- [The Truth Machine: The Blockchain and the Future of Everything](https://www.goodreads.com/en/book/show/34964890-the-truth-machine) _February 27, 2018 - Paul Vigna, Michael J. Casey_
- [Digital Gold: Bitcoin and the Inside Story of the Misfits and Millionaires Trying to Reinvent Money](https://www.goodreads.com/book/show/23546676-digital-gold) _May 24, 2021 - Nathaniel Popper_
- [Kings of Crypto: One Startup's Quest to Take Cryptocurrency Out of Silicon Valley and Onto Wall Street](https://www.goodreads.com/en/book/show/56274031-kings-of-crypto) _December 15, 2020 - Jeff John Roberts_
```

**Comment by minimalsm** [comment]
Awesome! Thanks for the PR @TrevorFrench 🙏

I've switched out the Amazon links for the corresponding Goodreads. Better experience for users who aren't based in the US ☺️ If you're good with that I can go ahead and get this merged 👍

**Comment by TrevorFrench** [comment]
@minimalsm No problem! Thanks for letting me contribute. 

I went ahead and committed your changes.

**Review by samajammin — APPROVED** [review]

**Review by corwintines — APPROVED** [review]

**Comment by minimalsm** [comment]
Thanks for this @TrevorFrench ✌️.

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And remember to [claim your POAP](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @TrevorFrench for content

**Pull request merged**